srcpkgs/xorg-server/template

```diff
@@ -109,6 +102,13 @@ xorg-server-xvfb_package() {
 	}
 }
 
+xorg-server-common_package() {
```

Comment:
If this is being split out, it should be in `depends` for `xorg-server`. But I'd like to confirm that this is necessary and it can't simply be removed from `Xwayland`, which would be much easier.

Reply:
Other distributions (arch, alpine) split protocol.txt to a separate package, but I tested if xwayland works without this file and it seems to work fine. The source seems to suggest protocol.txt is only installed and never used. I removed the subpackage.

Reply:
It seems to be used here: https://sources.debian.org/src/xorg-server/2:1.20.10-3/dix/registry.c/?hl=35#L35

Reply:
Oh, thanks for catching this. This code path is hit sometimes, because we enable xcsecurity, I will add the -common subpackage back.